### PR TITLE
Convert scripts to WP-CLI commands

### LINKED ISSUE #23: Convert scripts to WP-CLI commands
In order to unify, we should make these WP-CLI commands, rather than directly invoked scripts.

Unfortunately, WP-CLI doesn't support short options (ala `-o xyz`), so this needs a proper rewrite.

Working on this right now. :)

### PR DESCRIPTION
**Pull request by rmccue**
Fixes #23.

Most of the commands I've imported as essentially the same as previous, so that you can change from `php whatever-script.php` to `wp glotpress whatever-script`. The various translation set-related scripts (export, import, and recheck-warnings) are now subcommands of `wp glotpress translation-set` however.

Eventual plan should be to replace these legacy scripts completely, but I think they really need a proper WP-CLI-oriented rewrite using subcommands properly.

### CODE REVIEW

**Comment by polevaultweb** [comment]
@ocean90 needs wordpress-plugin-dbi merged in [gardening]

**Comment by polevaultweb** [comment]
@ocean90 needs wordpress-plugin-dbi merged in [gardening]

**Comment by bradt** [comment]
@rmccue Took a quick look and the approach looks good. Needs a merge before I can test though.

**Comment by rmccue** [comment]
Closing, it needs a weird merge, so will open a new (fresh) PR.

**Comment by rmccue** [comment]
Oh, I am incredibly dumb.

**Comment by bradt** [comment]
I will test next time I work on GP. If someone beats me to it, all the better.

File: gp-includes/cli/remove-multiple-currents.php [hunk 1]
```diff
@@ -11,7 +9,12 @@ function run() {
 			$prev_original_id = null;
 			foreach( $translations as $translation ) {
 				if ( $translation->original_id == $prev_original_id ) {
-					echo "Duplicate with original_id#$prev_original_id. Translation#$translation->id\n";
+					WP_CLI::line( sprintf(
+						/* translators: 1: original ID, 2: new ID */
+						__( "Duplicate with original_id#%s. Translation#%s\n", 'glotpress' )
```
Comment (ocean90) [inline gp-includes/cli/remove-multiple-currents.php]: Missing comma

File: gp-includes/cli/remove-multiple-currents.php [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
-require_once dirname( dirname( __FILE__ ) ) . '/gp-load.php';
 
-class GP_Script extends GP_CLI {
-
-	function run() {
+class GP_CLI_Remove_Multiple_Currents extends WP_CLI_Command {
+	public function __invoke() {
 		$sets = GP::$translation_set->all();
 		foreach( $sets as $set ) {
 			echo "Processing set#$set->id...\n";
```
Comment (ocean90) [inline gp-includes/cli/remove-multiple-currents.php]: Should use `WP_CLI::line()`

File: gp-includes/cli/translation-set.php [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+		$set_slug = isset( $assoc_args['set'] ) ? $assoc_args['set'] : 'default';
+		$translation_set = $this->get_translation_set( $args[0], $args[1], $set_slug );
+		if ( is_wp_error( $translation_set ) ) {
+			WP_CLI::error( $translation_set->get_error_message() );
+		}
+
+		$po = new PO();
+		$po->import_from_file( $args[2] );
+		$added = $translation_set->import( $po );
+		printf( _n( "%s translation were added\n", "%s translations were added\n", $added, 'glotpress' ), $added );
```
Comment (ocean90) [inline gp-includes/cli/translation-set.php]: Should use `WP_CLI::line()`

File: gp-includes/cli/wporg2slug.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+				$slug = $locale->slug;
+				break;
+			}
+		}
+
+		if ( ! $slug ) {
+			WP_CLI::error( sprintf( __( 'No slug match for %s.', 'glotpress' ), $wporg_slug ) );
+		}
+
+		echo $slug;
```
Comment (ocean90) [inline gp-includes/cli/wporg2slug.php]: Should use `WP_CLI::line()`

**Pull request merged**